instaclustr/api_client.go

```diff
@@ -286,6 +286,21 @@ func (c *APIClient) GCPReadVpcPeering(cdcID string, vpcPeeringID string) (*GCPVP
 	return &vpcPeering, nil
 }
 
+func (c *APIClient) AzureReadVpcPeering(cdcID string, vpcPeeringID string) (*AzureVPCPeering, error) {
+	url := fmt.Sprintf("%s/provisioning/v1/vpc-peering/%s/%s", c.apiServerHostname, cdcID, vpcPeeringID)
+	resp, err := c.MakeRequest(url, "GET", nil)
+	if err != nil {
+		return nil, err
+	}
+	bodyText, err := ioutil.ReadAll(resp.Body)
+	if resp.StatusCode != 202 {
+		return nil, errors.New(fmt.Sprintf("Status code: %d, message: %s", resp.StatusCode, bodyText))
+	}
+	var vpcPeering AzureVPCPeering
+	json.Unmarshal(bodyText, &vpcPeering)
```

Comment:
Can `json.Unmarshal` return an error? If so, it should be checked and handled properly